### PR TITLE
Updated Partners Page

### PR DESCRIPTION
**Pull request by hemanthReddy06**

### CODE REVIEW

File: dimagi/pages/templates/svg/partner/oikoi grey with presentation attributes.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (biyeun) [inline dimagi/pages/templates/svg/partner/oikoi grey with presentation attributes.html]: can you eliminate some of the extra tags and comments in here please? see the other SVG files for reference. Also please close all the tags e.g. `<circle></circle>` remove any `<g>` tags and make sure the `svg` tag contains only the same attributes as the other examples?

File: dimagi/pages/templates/svg/partner/oikoi.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 250 62" width="250px" height="62px">
+<g id="Bases">
+</g>
```
Comment (biyeun) [inline dimagi/pages/templates/svg/partner/oikoi.html]: cleanup svg

**Comment by hemanthReddy06** [comment]
@biyeun I have cleaned SVG and committed the changes.

**Comment by biyeun** [comment]
@hemanthReddy06

Thanks, changes look good. However...

> please make sure the `svg` tag contains only the same attributes as the other examples

This is still left to do. :-)

**Comment by hemanthReddy06** [comment]
@biyeun  I have compared with other partner SVG looks as I've made all changes, correct me if I missed anything.

**Comment by biyeun** [comment]
@hemanthReddy06 thanks, loooks good!

**Pull request merged**